### PR TITLE
Added assets and updated readme tested up to 4.8

### PR DESCRIPTION
**Pull request by manatarms**
Updated the assets and the readme

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -21,7 +21,8 @@
 ?>
 <div id="root" class="cloudflare-partners site-wrapper"></div>
 <script>
-var absoluteUrlBase = '<?php echo plugins_url('/cloudflare/'); ?>';
+//Set global absolute base url
+window.absoluteUrlBase = '<?php echo plugins_url('/cloudflare/'); ?>';
```
Comment (manatarms) [inline index.php]: This is the only legit coding change

**Comment by jwineman** [comment]
Wait to merge until we have the updated `compiled.js` here.

File: readme.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Contributors: jwineman, furkan811, icyapril
 Tags: cloudflare, seo, ssl, ddos, speed, security, cdn, performance, free
 Requires at least: 3.4
-Tested up to: 4.7.2
+Tested up to: 4.8
```
Comment (manatarms) [inline readme.txt]: This is the tested up to version update.

**Comment by thellimist** [comment]
For PR's like this try to to do different commits for stuff we can't review, and stuff we can review. For this one PR, assets could be one commit and `.php`, `.txt` files could be another commit.

**Comment by thellimist** [comment]
For compiled.js update 

I do a frontend release `npm run build:production`
https://github.com/cloudflare/cloudflare-plugin-frontend/releases

Then do a commit with the version
https://github.com/cloudflare/Cloudflare-WordPress/commit/160d606b8b9fe03fc36cab9d2aea9aa2d8ec6677

**Comment by manatarms** [comment]
@thellimist sorry about the single commit, I was feeling annoyed because nothing actually changed, github is diffing the content and git is diffing the meta data. I'm going to wait to PR the prod build till @jwineman tests all the integrations

**Comment by manatarms** [comment]
Built from https://github.com/cloudflare/cloudflare-plugin-frontend/commit/b4b4c72da6637b861cbf06e0991dd806c21baa5c

**Pull request merged**